### DISCUSSION TITLE
How To Expire Keys in Redis .

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#14038] [Q&A] [answered]
Hi. 
How To Expire Keys in Redis (setting TTL)  ? while my vector pipeline creates key and send my events to it.

**Comment by jszwedko** [comment, marked as answer]
Hi @LinTechSo ! There isn't currently a mechanism for setting the TTL on keys that Vector creates. If this would be useful to you, would you want to open a feature request with more details about your use-case? Like would you want Vector to set the TTL on any new keys it creates? Do you want it to reset the TTL on each insert operation?
